src/flows/components/AddButtons.tsx

```diff
@@ -80,15 +82,27 @@ const AddButtons: FC<Props> = ({index, onInsert}) => {
     return acc
   }, {})
 
+  const pipeIds = flow.data.allIDs
+  const firstQueryIndex = pipeIds.findIndex(
```

Comment:
a bit torn about this. seems like it should be run at point of consumption, but maybe this is fine